### DISCUSSION TITLE
Canales en SD

### DISCUSSION THREAD

**Discussion by granavi** [Paco8/plugin.video.movistarplus#42] [General]
Hola
Lo primero felicitarte por esta maravilla que acabo de descubrir. Y quisiera hacerte una pregunta , me salen todos los canales en SD ¿es esta la resolución máxima que se puede tener?
Instalado en un firetv con Kodi 21.

Gracias por tu trabajo
Un saludo

**Comment by Paco8** [comment]
Mira la configuración de inputstream adaptive, puede que te esté limitando la resolución.

**Reply by granavi** [reply]
Hola
Si era eso, muchas gracias
